### DISCUSSION TITLE
Using Xamarin.Firebase.iOS.CloudMessaging and Uno.winui.Lottie together causes a crash.

### DISCUSSION THREAD

**Discussion by EricKwon94** [unoplatform/uno#14009] [Q&A]
### Current behavior

When trying to build on a real iPhone device after pairing with a Mac in Windows Visual Studio, it crashes as follows.

/Users/runner/.nuget/packages/skiasharp.nativeassets.ios/2.88.1-preview.79/buildTransitive/net6.0-ios13.6/SkiaSharp.NativeAssets.iOS.targets(6,9): error MSB4096: The item "/Users/runner/Library/Caches/XamarinBuildDownload/GAppM-8.9.1/GoogleAppMeasurement-8.9.1/Frameworks/GoogleAppMeasurement.xcframework" in item list "NativeReference" does not define a value for metadata "FrameworkFilename".  In order to use this metadata, either qualify it by specifying %(NativeReference.FrameworkFilename), or ensure that all items in this list define a value for this metadata. [/Users/runner/work/Test/Test.csproj::TargetFramework=net7.0-ios]

### Works on UWP/WinUI

No

### Environment

Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia

### NuGet package version(s)

Xamarin.Firebase.iOS.CloudMessaging 8.10.0.3
Uno.WinUI.Lottie 4.10.13

### Affected platforms

iOS

### IDE

Visual Studio 2022

### IDE version

17.7.2

**Comment by jeromelaban** [comment]
Thanks for the report! You seem to be using an older version of SkiaSharp, you may want to update to the latest stable builds and try again.

**Reply by EricKwon94** [reply]
The SkiaSharp library is not installed in my project. Even after installing Uno 4.10.26, the error persists. It's important to note that the build works fine when Xamarin.Firebase.iOS.CloudMessaging is removed.

**Reply by jeromelaban** [reply]
The skiasharp library is installed automatically when referencing the Lottie packages. You'll need to install an explicit reference to the latest version (2.88.6) and see if this helps.
